data/random-teams.ts

```diff
@@ -1990,7 +1992,7 @@ export class RandomTeams {
 			if (!species.exists) continue;
 
 			// AFD enforcement of at least 1 CAP pokemon per team
-			if (!isMonotype && pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
+			if (pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
```

Comment:
```suggestion
			if (!isMonotype && pokemon.length === 5 && !hasCAP && species.isNonstandard !== 'CAP') continue;
```
this is necessary, there arent enough CAPs to make monotype teams with them enforced

Reply:
i didn't do this on purpose, this is just a conflict with your other thing